Process.h

```diff
@@ -160,7 +161,10 @@ typedef struct Process_ {
    /* Scheduling priority */
    long int priority;
 
-   /* Nice value */
+   /* Nice value
+    *
+    * PROCESS_NICE_UNKNOWN is a sentinel value for unknown niceness */
+#define PROCESS_NICE_UNKNOWN (-LONG_MAX)
```

Comment:
Minor style thing: I think we should avoid having `#define`s intermixed with `struct` declarations. I see two possible places where this could go: Near the top before the define for `DEFAULT_HIGHLIGHT_SECS` or down below this structure; I'd probably prefer the first location.